src/transformers/models/swin2sr/modeling_swin2sr.py

```diff
@@ -301,6 +301,9 @@ def __init__(self, config, dim, num_heads, window_size, pretrained_window_size=[
         relative_coords_table = (
             torch.sign(relative_coords_table) * torch.log2(torch.abs(relative_coords_table) + 1.0) / math.log2(8)
         )
+        relative_coords_table = relative_coords_table.to(
+            next(self.continuous_position_bias_mlp.parameters()).dtype
+        )  # set to same dtype as mlp weight
```

Comment:
nit - library convention is to place comments above to avoid line splitting (I realise this is somewhat violated above)

```suggestion
        # set to same dtype as mlp weight
        relative_coords_table = relative_coords_table.to(next(self.continuous_position_bias_mlp.parameters()).dtype)
```